onnxruntime/core/optimizer/conv_add_act_fusion.cc

```diff
@@ -69,7 +71,7 @@ class ConvAddActivationSelector : public NodeSelector {
     // 1 find the Add node, 2 find it's producer node and make sure it's a conv node
     // 3 find the next node and check if it's a activation node, if yes, we will fuse conv+add+activation or conv+add
     //
-    if (graph_utils::IsSupportedOptypeVersionAndDomain(*add_node, "Add", {7, 13, 14})) {
+    if (graph_utils::IsSupportedOptypeVersionAndDomain(*add_node, "Add", {7, 13, 14}, node.Domain())) {
```

Comment:
we're only looking for ONNX Add right? do we need to pass in `node.Domain()` here?

Reply:
Removed node.Domain()